src/components/form/TextInputField.tsx

```diff
@@ -0,0 +1,19 @@
+import { FunctionComponent } from "react";
+import { Field, FieldProps } from "formik";
+import { TextInput, TextInputProps } from "elements";
+
+const TextInputField: FunctionComponent<TextInputProps> = (props) => {
```

Comment:
I was a bit conflicted about where to put it. I ended up putting it in the components directory just because it seemed a bit more like a composite component, but I think either directory would be valid.